tutorials/notebooks/imx500_notebooks/pytorch/pytorch_mobilevit_xs_for_imx500.ipynb

```diff
@@ -215,7 +215,8 @@
     "                                           target_platform_name='imx500',\n",
     "                                           target_platform_version='v1')\n",
     "ptq_config = mct.core.CoreConfig(quantization_config=mct.core.QuantizationConfig(\n",
-    "    shift_negative_activation_correction=True))\n",
```

Comment:
Please add a short comment to explain the z_threshold.. maybe mention that it's a general rule of thumb for transformers architecture.